t/t2204-job-info.t

```diff
@@ -102,6 +102,29 @@ test_expect_success 'load job-exec,sched-simple modules' '
 #   unordered fashion.  However, issues exist that do not allow for
 #   such testing at this moment.  See Issue #2470.
 # - alternate userid job listing
+#
+# the job-info module has eventual consistency with the jobs stored in
+# the job-manager's queue.  To ensure no raciness in tests, we spin
+# until all of the pending jobs have reached SCHED state, running jobs
+# have reached RUN state, and inactive jobs have reached INACTIVE
+# state.
+
+wait_states() {
+        local i=0
+        while ( [ "$(flux job list -A | jq .state | grep 4 | wc -l)" != "4" ] \
+                || [ "$(flux job list -A | jq .state | grep 8 | wc -l)" != "8" ] \
+                || [ "$(flux job list -A | jq .state | grep 32 | wc -l)" != "4" ]) \
```

Comment:
Would it be better/simpler here to do `$(flux job list --states=sched | wc -l)` `$(flux job list --states=run | wc -l)` and `$(flux job list --states=inactive | wc -l)` to explicitly count the jobs in these states?

Or would `flux job stats` work?

Reply:
Ahhh, it would be better!  Good catch.